fetch.bs

```diff
@@ -4459,6 +4438,35 @@ the response. [[!HTTP-CACHING]]
 </ol>
 </div>
 
+<div algorithm>
+<p>To <dfn>populate request from client</dfn> given a <a for=/>request</a> <var>request</var>:
+
+<ol>
+ <li><p>If <var>request</var>'s <a for=request>window</a> is "<code>client</code>", then: set
+ <var>request</var>'s <a for=request>window</a> to <var>request</var>'s <a for=request>client</a>
+ if <var>request</var>'s <a for=request>client</a>'s
+ <a for="environment settings object">global object</a> is a {{Window}} object; otherwise
+ "<code>no-window</code>".
+
+ <li><p>If <var>request</var>'s <a for=request>origin</a> is "<code>client</code>", then set
+ <var>request</var>'s <a for=request>origin</a> to  <var>request</var>'s <a for=request>client</a>'s
+ <a for="environment settings object">origin</a>.
+
+ <li>
+  <p>If <var>request</var>'s <a for=request>policy container</a> is "<code>client</code>":
+
+  <ol>
+   <li><p>If <var>request</var>'s <a for=request>client</a> is non-null, then set
+   <var>request</var>'s <a for=request>policy container</a> to a
+   <a lt="clone a policy container">clone</a> of <var>request</var>'s <a for=request>client</a>'s
+   <a for="environment settings object">policy container</a>. [[!HTML]]
+
+   <li><p>Otherwise, set <var>request</var>'s <a for=request>policy container</a> to a new
+   <a for=/>policy container</a>.
+  </ol></div>
```

Comment:
```suggestion
  </ol>
```
I'm surprised this is not a build error. I guess Bikeshed parsing just drops it? cc @tabatkins